packages/react/src/radio-group/RadioGroup.tsx

```diff
@@ -13,12 +13,26 @@ const mapDirectionToOrientation = {
   row: "horizontal",
 } as const;
 
+const mapGapToOrientation = {
+  column: "sm",
+  row: "md",
+} as const;
+
 export const RadioGroup = forwardRef<HTMLDivElement, RadioGroupProps>(
   ({ children, flexDirection = "column", ...props }, ref) => {
     const { restProps, sprinkleProps } = extractSprinkles(props);
 
     return (
-      <Flex asChild flexDirection={flexDirection} gap="sm" {...sprinkleProps}>
+      <Flex
+        asChild
+        flexDirection={flexDirection}
+        gap={
+          flexDirection === "row" || flexDirection === "column"
```

Comment:
We have to use `mapResponsiveValue` here instead of just a simple map

Example:

https://github.com/optimizely-axiom/optiaxiom/blob/1a85ead9590fbe71af505616c0b1192cf0665ef3/packages/react/src/flex/Flex.tsx#L37-L40